package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ballantine-adult-fantasy-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./config/sequelize.js",
+    "eslint": "./node_modules/.bin/eslint --format codeframe .",
+    "lint": "npm run lint:puglint && npm run lint:eslint",
```

Comment:
Whole bunch of lint errors

![image](https://user-images.githubusercontent.com/5775716/83886532-2da28980-a715-11ea-8b99-1a158adb5067.png)